### DISCUSSION TITLE
Inserting an image into Modelica documentation browser

### DISCUSSION THREAD

**Discussion by MohammadBabaeifar** [OpenModelica/OpenModelica#10841] [Q&A] [answered]
Hi,
I am trying to insert an image into OpenModelica in its documentation browser. My image is in .png format and is named template.png. It is located in the root of D directory which is D:\
So, in the info source, where it is possible to write HTML codes, I wrote this code: 
```html
<html>
<img src="template.png" alt="template">
</html>
```
However, what I see then is only a question mark.
Could you please help how know that how I can insert my image?

**Comment by casella** [comment, marked as answer]
@MohammadBabaeifar the right way to proceed is the following:
- save all of your models in a package
- map the package in a directory, not in a single file
- Put a Resources/Images directory in the directory that contains your package

Files in there can then be accessed via URI, e.g. modelica://MyLibrary/Resources/Images/xxx.png

You can check e.g. the [PowerGrids](https://github.com/PowerGrids/PowerGrids) library to see how this is accomplished.

**Reply by MohammadBabaeifar** [reply]
Dear Professor Casella,
Thank you for your help. I tried it now successfully.
